### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
----

### Before the change?

* typos https://github.com/jsoref/go-sdk/actions/runs/7951952434#summary-21705988722

### After the change?

* https://github.com/jsoref/go-sdk/actions/runs/7951952779#summary-21705989302

### Pull request checklist
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)

### Does this introduce a breaking change?

Please see our docs on [breaking changes](https://github.com/octokit/.github/blob/master/community/breaking_changes.md) to help!

- [ ] Yes
- [ ] No

----

### CODE REVIEW

File: pkg/github/gists/with_gist_item_request_builder.go [hunk 1]
```diff
@@ -87,7 +87,7 @@ func (m *WithGist_ItemRequestBuilder) Get(ctx context.Context, requestConfigurat
...
+// Patch allows you to update a gist's description and to update, delete, or rename gist files. Files from the previous version of the gist that aren't explicitly changed during an edit are unchanged. At least one of `description` or `files` is required.This endpoint supports the following custom media types. For more information, see "[Media types](https://docs.github.com/rest/using-the-rest-api/getting-started-with-the-rest-api#media-types)."- **`application/vnd.github.raw+json`**: Returns the raw markdown. This is the default if you do not pass any specific media type.- **`application/vnd.github.base64+json`**: Returns the base64-encoded contents. This can be useful if your gist contains any invalid UTF-8 sequences.
```
Comment (jsoref) [inline pkg/github/gists/with_gist_item_request_builder.go:90]: There are *lots* of missing spaces in these items. I really don't understand how this happened.

Even in this entry there's another space that's missing:
`required.This`

File: pkg/github/models/code.go [hunk 2]
```diff
@@ -16,7 +16,7 @@ type Code struct {
     object_type *string
     // The object_url property
     object_url *string
-    // The property property
+    // The property
```
Comment (jsoref) [inline pkg/github/models/code.go:19]: I'm happy to drop this change... _generally_ duplicated words are errors, and often the proper fix is additional punctuation.

File: pkg/github/models/converted_note_to_issue_issue_event.go [hunk 3]
```diff
@@ -4,7 +4,7 @@ import (
     i878a80d2330e89d26896388a3f487eef27b0a0e6c010c493bf80be1452208f91 "github.com/microsoft/kiota-abstractions-go/serialization"
 )
 
-// ConvertedNoteToIssueIssueEvent converted Note to Issue Issue Event
+// ConvertedNoteToIssueIssueEvent converted Note to Issue IssueEvent
```
Comment (jsoref) [inline pkg/github/models/converted_note_to_issue_issue_event.go:7]: Here, the `converted Note to Issue` is an `IssueEvent` and it makes sense to make it possible to parse by removing the space ...

File: pkg/github/orgs/item_actions_oidc_customization_sub_request_builder.go [hunk 4]
```diff
@@ -23,7 +23,7 @@ func NewItemActionsOidcCustomizationSubRequestBuilder(rawUrl string, requestAdap
     urlParams["request-raw-url"] = rawUrl
     return NewItemActionsOidcCustomizationSubRequestBuilderInternal(urlParams, requestAdapter)
 }
-// Get gets the customization template for an OpenID Connect (OIDC) subject claim.You must authenticate using an access token with the `read:org` scope to use this endpoint.GitHub Apps must have the `organization_administration:write` permission to use this endpoint.
+// Get gets the customization template for an OpenID Connect (OIDC) subject claim.You must authenticate using an access token with the `read:org` scope to use this endpoint. GitHub Apps must have the `organization_administration:write` permission to use this endpoint.
```
Comment (jsoref) [inline pkg/github/orgs/item_actions_oidc_customization_sub_request_builder.go:26]: `claim.You` should also be split...

**Comment by nickfloyd** [comment]
Hey @jsoref thank you for the work here!  The files that you changed are generated.  So if these changes were applied the would be overwritten the next time generation ran.  You can submit an [issue to the descriptions repo ](https://github.com/github/rest-api-description/issues)so that the teams who own the OpenAPI descriptions can make the changes you're suggesting. 

Sorry for the trouble here - but if you link your PR here to your issue then the teams will be able to get these changes in permanently.

**Comment by kfcampbell** [comment]
Hi @jsoref, thanks for the interest! For this project, please keep changes out of the `pkg/github` directory, since those are generated from GitHub's OpenAPI spec and any manual fixes would be wiped out on the next generation. I'm going to close this PR but please feel free to follow up if you have any questions.

**Pull request closed without merging**